docs/reference/contrib/settings.md

```diff
@@ -148,6 +146,8 @@ def view(request):
     ...
 ```
 
+(site_settings)=
```

Comment:
Nice one! 

There is a 'site settings' link on https://docs.wagtail.org/en/stable/advanced_topics/multi_site_multi_instance_multi_tenancy.html that now links to this section thanks to this change 👍

Reply:
yeah - I forgot we have now got generic settings as of 4.0 yay